Src/LabHostConfiguration.ps1

```diff
@@ -155,10 +155,26 @@ function Start-LabHostConfiguration {
 		$hostDefaults = GetConfigurationData -Configuration Host;
         foreach ($property in $hostDefaults.PSObject.Properties) {
             if (($property.Name.EndsWith('Path')) -and (-not [System.String]::IsNullOrEmpty($property.Value))) {
-                [ref] $null = NewDirectory -Path $property.Value -ErrorAction Stop;
+                [ref] $null = NewDirectory -Path $(ResolvePathEx -Path $Property.Value) -ErrorAction Stop;
             }
         }
         
+        # Once all the path are created, check if the hostdefaults.Json file in the $env:ALLUSERSPROFILE is doesn't have entries with %SYSTEMDRIVE% in it
+        # Many subsequent call are failing to Get-LabImage, Test-LabHostConfiguration which do not resolve the "%SYSTEMDRIVE%" in the path for Host defaults
+        foreach ($property in $($hostDefaults.PSObject.Properties | where -Property TypeNameOfValue -ne 'System.Boolean')) {
+            if ($property.Value.Contains("%")){
+                # if the Path for host defaults contains a '%' character then resolve it
+                $resolvedPath = ResolvePathEx -Path $Property.Value
+                # update the hostdefaults Object
+                $hostDefaults.($property.name)  = $resolvedPath
+                $hostdefaultupdated = $True
+            }
+        }
+        if ($hostdefaultupdated) {
+            # Write the changes back to the json file in the $env:ALLUSERSPROFILE
```

Comment:
@iainbrighton added the logic to update the hostdefaults.json placed under the $env:ALLUSERSPROFILE after resolving the paths having environment varaiables.